src/main/java/net/aufdemrand/denizen/events/player/PlayerItemTakesDamageScriptEvent.java

```diff
@@ -125,5 +125,6 @@ public void onPlayerItemTakesDamage(PlayerItemDamageEvent event) {
         fire();
         event.setCancelled(cancelled);
         event.setDamage(damage.asInt());
+        event.getPlayer().updateInventory();

```

Comment:
To clarify, have you checked with vast amounts and visually confirmed? As in, changing the damage to 50% of the durability, but also cancelling it to make 100% sure etc